modules/post/multi/gather/grub_password.rb

```diff
@@ -0,0 +1,61 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+  include Msf::Post::Unix
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'         => '*nix Gather Grub Password',
+      'Description'  => %q{
+        This module gathers grub passwords from grub bootloader config file.
+      },
+      'License'      => MSF_LICENSE,
+      'Author'       =>
+        [
+          'Garvit Dewan <d.garvit[at]gmail.com>' # @dgarvit
+        ],
+      'Platform'     => ['linux', 'osx', 'unix', 'solaris', 'bsd'],
```

Comment:
Also, I wanted to confirm if OSX uses GRUB. https://superuser.com/questions/606898/how-to-install-grub-command-line-tools-on-osx/1004171 says that it is possible, that is why I added it.

Reply:
Some googling indicates that OSX is a valid target, and that `/boot/grub/grub.cfg` (already in the `targets` list) might be a default.